module/VuFind/src/VuFind/Controller/CombinedController.php

```diff
@@ -225,6 +225,17 @@ public function resultsAction()
         $settings = $this->serviceLocator->get(\VuFind\Config\PluginManager::class)
             ->get('config');
 
+        // Identify if any modules use include_recommendations_side.
+        $columnSideRecommendations = [];
+        $recommendationManager = $this->serviceLocator->get(\VuFind\Recommend\PluginManager::class);
+        foreach ($config as $subconfig) {
+            foreach ($subconfig['include_recommendations_side'] ?? [] as $recommendation) {
```

Comment:
Maybe worth making this `(array)($subconfig['include_recommendations_side'] ?? [])` so if somebody adds a single recommendation module without the brackets, it doesn't break.

Reply:
I'd rather not do this one -- I'd have to also change [line 350](https://github.com/vufind-org/vufind/pull/3135/files#diff-0a4a3ff1ed93bd3cbe0a7eea71dc6dbd83c272d49b0c87f84f3ef56716c32c25R350) and that would make it inconsistent with `include_recommendations`.

Reply:
That works!  Done.